frontend/src/modules/entitieslist/components/EntitiesList.tsx

```diff
@@ -219,10 +249,15 @@ export class EntitiesListBase extends React.Component<InternalProps> {
 
     getSiblingEntities: (entity: number) => void = (entity: number) => {
         const { dispatch, locale } = this.props;
-
         dispatch(entities.actions.getSiblingEntities(entity, locale.code));
     };
 
+    // need to be able to remove siblings when an new parent is selected
+    removeSiblingEntities: () => void = () => {
+        console.log('*** we are removing siblings');
+        this.props.dispatch(entities.actions.reset());
```

Comment:
I don't think we want this. I couldn't test it, because all entities get reloaded when you switch entities, but I assume it would be unexpected and confusing.